src/runtime/propensity-server.js

```diff
@@ -39,6 +48,16 @@ export class PropensityServer {
     this.xhr_ = new Xhr(win);
     /** @private @const {number} */
     this.version_ = 1;
+
+    eventManager.registerEventListener(event => this.handleClientEvent_(event));
+
+    // TODO(mborof): b/133519525
+    /** @private @const {!boolean} */
```

Comment:
I think we can keep it as this.logSwgEvents_ = isExperimentOn(...)

and then and it with the enable logging flag. No?

Any reason we need to keep track of two flags?